glyphs-reader/src/font.rs

```diff
@@ -2584,7 +2586,11 @@ fn preprocess_unparsed_plist(s: &str) -> Cow<str> {
 impl Font {
     pub fn load(glyphs_file: &path::Path) -> Result<Font, Error> {
         let mut font = Self::load_raw(glyphs_file)?;
-        font.propagate_all_anchors();
+        if font.custom_parameters.propagate_anchors.is_none()
+            || font.custom_parameters.propagate_anchors == Some(true)
+        {
+            font.propagate_all_anchors();
+        }
```

Comment:
Here I used "is none or true" because these are the two cases where it should propagate anchors:
* When the custom param is missing (defaults to true)
* Or when it is explicitly true

But I think it may be simpler to simply do `if font.custom_parameters.propagate_anchors != Some(false)` instead...